py3status/modules/window_title.py

```diff
@@ -51,6 +51,9 @@ def window_title(self):
 
         transformed = False
         if window and 'name' in window and window['name'] != self.text:
```

Comment:
This will end up triggering when `window['name'] == None`.  Anyhow I'm not sure that this is an issue, more observation, so I'm happy with this to stay as it is.

To be honest I'm not why we are returning the transformed to i3bar it does not seem to be part of the protocol.

@ultrabug do you know anything about the transformed stuff?